components/arm/fake/fake.go

```diff
@@ -24,6 +24,16 @@ import (
 	"go.viam.com/rdk/spatialmath"
 )
 
+// errModelName is the returned string when we want to use a path to
+// instantiate a model.
+const errModelName = "fake arm cannot be created, unsupported arm_model: "
+
+// errAttrCfgPopulation is the returned string if the AttrConfig's fields are fully populated.
+const errAttrCfgPopulation = "can only populate either ArmModel or ModelPath - not both"
+
+// errAttrCfgMissing is the returned string if the AttrConfig's fields are empty.
+const errAttrCfgMissing = "one of ArmModel or ModelPath must be populated"
+
 // ModelName is the string used to refer to the fake arm model.
 var ModelName = resource.NewDefaultModel("fake")
```

Comment:
this will become accessible if someone requests a "fake" arm.  I thought that the empty string was going to denote a fake arm?

Reply:
We've overloaded the meaning of "model" at Viam.

This doesn't refer to the frame model. This is the string used to register the component so that you can refer to "this thing" from your RDK config.

Discussion so far, including from product, has been around changing the parameters of the fake arm and what values for them mean. To make this an empty string is likely invalid in RDK.

Reply:
Sorry this was not clear, I made another comment explaining what I mean